src/Block/Service/TextBlockService.php

```diff
@@ -29,10 +29,7 @@
  */
 final class TextBlockService extends AbstractAdminBlockService
 {
-    /**
-     * {@inheritdoc}
-     */
-    public function execute(BlockContextInterface $blockContext, Response $response = null)
+    public function execute(BlockContextInterface $blockContext, Response $response = null): Response
```

Comment:
```suggestion
    public function execute(BlockContextInterface $blockContext, ?Response $response = null): Response
```

Also for the other blocks...